$OpenBSD: patch-src_QGlib_value_h,v 1.1 2017/12/26 19:18:57 rsadowski Exp $

Workaround for boost>=1.57 and MOC

The workaround to define BOOST_TT_HAS_OPERATOR_HPP_INCLUDED
doesn't work anymore.

https://bugreports.qt-project.org/browse/QTBUG-22829 for details.

Index: src/QGlib/value.h
--- src/QGlib/value.h.orig
+++ src/QGlib/value.h
@@ -23,8 +23,10 @@
 #include "type.h"
 #include "refpointer.h"
 #include "error.h"
+#ifndef Q_MOC_RUN
 #include <boost/mpl/if.hpp>
 #include <boost/type_traits.hpp>
+#endif
 #include <stdexcept>
 #include <QtCore/QString>
 #include <QtCore/QDebug>
